libs/sam/src/build/get-entries-from-cloudformation.ts

```diff
@@ -16,21 +16,24 @@ export function getEntriesFromCloudFormation(
     context: BuilderContext
 ): Array<Entry> {
     const cf = loadCloudFormationTemplate(options.template);
+    // @ts-ignore
```

Comment:
Can you amend the return type in `loadCloudFormationTemplate` instead of using `@ts-ignore`?

Reply:
totally agree.  i've added a new lib to hold types that aren't supported in the upstream cloudform repo. when they're added, it'll be a case of removing the additional types module, with hopefully minimal modification to `nx-aws/sam`